meteor/sofie-server/data-access/repositories/mongo/mongo-database.ts

```diff
@@ -0,0 +1,54 @@
+import * as mongodb from 'mongodb'
+import { Collection } from 'mongodb'
+import { DatabaseNotConnectedException } from '../../../model/exceptions/database-not-connected-exception'
+
+// TODO: Move to ENV variables
+const MONGO_CONNECTION_STRING: string = 'mongodb://localhost:3001'
+const MONGO_DB_NAME: string = 'meteor'
+
+export class MongoDatabase {
+	private static instance: MongoDatabase
+
+	public static getInstance(): MongoDatabase {
+		if (!this.instance) {
+			this.instance = new MongoDatabase()
+		}
+		return this.instance
+	}
+
+	private client: mongodb.MongoClient
+	private db: mongodb.Db
+
+	private constructor() {
+		// eslint-disable-next-line @typescript-eslint/no-floating-promises
+		this.connectToDatabase().then()
```

Comment:
Is the `.then()` needed?

Reply:
Technically not.
If it's not there lint complains about missing await for a Promise, but we can't have an async constructor, so await is not an option.
But lint complains that we have an empty `then()` so I guess we might as well just ignore the "missing await" rule instead

Reply:
Might be a dumb way to "fix" the complaint, but adding a small log statement in `.then()` could be of use, while getting rid of the complaint.